### DISCUSSION TITLE
Right to left class

### DISCUSSION THREAD

**Discussion by 1mehdifaraji** [tailwindlabs/tailwindcss#12114] [Ideas]
There isn't a `rtl` class name to align elements from right to left . Tailwind only has `rtl:` which isn't applying right to left css style . I need `direction: right-to-left` css style .

**Comment by wongjn** [comment]
You could consider applying `display: flex; flex-direction: row-reverse` via `flex flex-row-reverse`: https://play.tailwindcss.com/gVOtvSMwNq

**Reply by 1mehdifaraji** [reply]
It won't work for arabic or persian text right to left styling . I need `rtl` `direction: right-to-left` css style .

**Reply by wongjn** [reply]
You could consider using an [arbitrary property class](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) like `[direction:rtl]` or, as per [the MDN documentation for `direction`](https://developer.mozilla.org/en-US/docs/Web/CSS/direction):

> ⚠️ Warning: Where possible, authors are encouraged to avoid using the `direction` CSS property and use the HTML [`dir`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#dir) global attribute instead.

https://play.tailwindcss.com/0EYYwBKimI

**Comment by lukewarlow** [comment]
https://github.com/tailwindlabs/tailwindcss/pull/6359 I attempted to add utilities for this. Currently the recommendation is to use arbitrary values (or a simple plugin to add your own utilities)
